Content.Shared/Nutrition/EntitySystems/HungerSystem.cs

```diff
@@ -44,6 +44,13 @@ public override void Initialize()
 
     private void OnMapInit(EntityUid uid, HungerComponent component, MapInitEvent args)
     {
+        // <goobstation> Starting hunger override
+        if (component.StartingHunger is not null)
+        {
+            SetHunger(uid, (float) component.StartingHunger, component);
```

Comment:
```suggestion
            SetHunger(uid, component.StartingHunger.Value, component);
```
Use Value instead of casting.

Reply:
True, got dementia'd there for a sec